iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -74,11 +68,9 @@ namespace Flow {
 static void buildHLOInputTransformPassPipeline(OpPassManager &passManager) {
   passManager.addNestedPass<FuncOp>(
       IREE::Flow::createHLOToHLOPreprocessingPass());
-  if (clEnableLinalgOnTensorsDispatch) {
-    // TODO(ataei): This should run as part of createHLOToHLOPreprocessingPass
-    // which will break VMLA backend.
-    passManager.addNestedPass<FuncOp>(createDecomposeHLOClampPass());
-  }
+  // TODO(ataei): This should run as part of createHLOToHLOPreprocessingPass
```

Comment:
Does this not break VMLA backend?

Reply:
It use to, not all HLO ops here has VMLA equivalent

Reply:
No it does not.